templates/includes/pagination.html

```diff
@@ -1,5 +1,6 @@
 {% if is_paginated %}
-  <ul class="pagination">
+  <hr class="mt-5">
+  <ul class="pagination justify-content-center">
     {% if page_obj.has_previous %}
       <li class="page-item">
         <a href="?page= {{ page_obj.previous_page_number }}" class="page-link">prev</a>
```

Comment:
There is an extra space after the '=' in the href attribute for the 'prev' link. It should be `href="?page={{ page_obj.previous_page_number }}"` to ensure the URL is constructed correctly.